include/deal.II/lac/read_write_vector.h

```diff
@@ -50,6 +53,14 @@ namespace TrilinosWrappers
     class Vector;
   }
 }
+
+namespace LinearAlgebra
+{
+  namespace EpetraWrappers
```

Comment:
I wouldn't mind shortening the name to ```Epetra``` because I thought the ```TrilinosWrappers::MPI::Vector``` was awfully long. Thoughts?